Kong/docs.konghq.com
# [DOCU-2077] Deprecate cookie policy banner in favor of Onetrust

### Summary
DO NOT MERGE UNTIL ONETRUST IS IMPLEMENTED

Removing the old docs.konghq.com cookie policy banner. 
Adding script that calls OneTrust cookie pop-up.

### Reason
This is being replaced by OneTrust, which provides much better & accurate opt-in/opt-out mechanisms.

### Testing
To test that the old banner is gone:
1. Using Chrome, open https://deploy-preview-3468--kongdocs.netlify.app in an incognito tab
2. Make sure that the old cookie policy banner does not appear at the bottom right of the page
3. Open any topic, eg https://deploy-preview-3468--kongdocs.netlify.app/gateway
4. Click the `>` next to the right side ("Edit this page") navbar to collapse it
5.  Right click anywhere on the page and select "Inspect"
6. Navigate to the "Application" tab
7. In the left menu, in Storage > Cookies, click on "deploy-preview". In the list that appears:
  * There should be **no** `cookiepolicy` cookie
  * There should be a `toc-collapsed` cookie visible
  
![Screen Shot 2021-12-09 at 9 33 44 AM](https://user-images.githubusercontent.com/54370747/145446840-ef5d76ed-ffac-4a02-b682-a5b0bbc1688b.png)

To test the new OneTrust banner: 
* In the bottom left corner of any preview page, the new cookie banner should pop up with two options: an "Accept all cookies" button and a link to "Do not sell my personal information". 
  * If you click the link below the button, you get a bunch of legal info and a toggle to turn off personal data selling
    Note: clicking the X instead of the link should also do the same thing - ie, opt out of selling data.
